### PR TITLE
[CMOB 1305]/Feature/design

### PR DESCRIPTION
**Pull request by DavitTsiskarashvili**
https://spaceneobank.atlassian.net/browse/CMOB-1305

### CODE REVIEW

**Comment by GiorgiMamagulashvil1** [comment]
change base branch, set a develop

File: app/src/main/java/com/example/spacechatapp/MainActivity.kt [hunk 1]
```diff
@@ -2,10 +2,43 @@ package com.example.spacechatapp
...
+        binding.swDayNight.setOnCheckedChangeListener { _, isChecked ->
+            if (isChecked) {
+                AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
+            } else {
+                AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
+            }
+        }
+    }
+
+}
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/java/com/example/spacechatapp/MainActivity.kt]: u can write something like, AppCompatDelegate.setDefaultNightMode( if() code else code ), it is more readable and reduces code repeats

File: app/src/main/res/drawable/ic_toggle_night_list.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <layer-list>
+            <item
+                android:bottom="3dp"
+                android:end="3dp"
+                android:start="3dp"
+                android:top="3dp"
+                android:drawable="@drawable/ic_toggle_night"/>
+        </layer-list>
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/res/drawable/ic_toggle_night_list.xml]: use resource dp s

File: app/src/main/res/layout/fragment_bottom.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+
+    <androidx.appcompat.widget.LinearLayoutCompat
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginHorizontal="@dimen/dimen_16_dp"
+        android:layout_marginBottom="@dimen/dimen_30_dp"
+        android:paddingEnd="@dimen/dimen_12_dp"
+        android:paddingStart="@dimen/dimen_12_dp"
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/res/layout/fragment_bottom.xml]: use marginHorizontal

File: app/src/main/res/layout/fragment_bottom.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+        <androidx.appcompat.widget.AppCompatEditText
+            android:id="@+id/etMessage"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:minHeight="@dimen/dimen_56_dp"
+            android:layout_weight="1"
+            android:background="@null"
+            android:paddingTop="12dp"
+            android:paddingBottom="12dp"
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/res/layout/fragment_bottom.xml]: use margin vertical and use resource dps

File: app/src/main/res/layout/fragment_bottom.xml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            android:scrollbars="vertical"
+            android:fadeScrollbars="false"
+            android:lineSpacingExtra="@dimen/dimen_4_sp"
+            android:scrollbarStyle="insideInset"
+            android:scrollbarThumbVertical="@drawable/ic_scroll_thumb"
+            android:scrollbarTrackVertical="@drawable/ic_scroll_track"
+            android:layout_marginStart="@dimen/dimen_22_dp"/>
+
+        <androidx.appcompat.widget.AppCompatImageButton
+            android:id="@+id/ibSend"
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/res/layout/fragment_bottom.xml]: very unreadable id name

File: app/src/main/res/layout/fragment_top.xml [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+
+    <androidx.appcompat.widget.LinearLayoutCompat
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginHorizontal="@dimen/dimen_16_dp"
+        android:layout_marginBottom="@dimen/dimen_30_dp"
+        android:paddingEnd="@dimen/dimen_12_dp"
+        android:paddingStart="@dimen/dimen_12_dp"
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/res/layout/fragment_top.xml]: marginHorizontal

**Review by GiorgiMamagulashvil1 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_bottom.xml (hunk 4 above)
Comment (DavitTsiskarashvili) [reply app/src/main/res/layout/fragment_bottom.xml]: I need padding so the text doesn't come close to the borderline of the editText, but I'll use dimens

File: app/src/main/res/layout/fragment_bottom.xml (hunk 3 above)
Comment (DavitTsiskarashvili) [reply app/src/main/res/layout/fragment_bottom.xml]: done

File: app/src/main/res/drawable/ic_toggle_night_list.xml (hunk 2 above)
Comment (DavitTsiskarashvili) [reply app/src/main/res/drawable/ic_toggle_night_list.xml]: done

File: app/src/main/java/com/example/spacechatapp/MainActivity.kt (hunk 1 above)
Comment (DavitTsiskarashvili) [reply app/src/main/java/com/example/spacechatapp/MainActivity.kt]: done

File: app/src/main/res/layout/fragment_bottom.xml (hunk 5 above)
Comment (DavitTsiskarashvili) [reply app/src/main/res/layout/fragment_bottom.xml]: Done. Renamed it ibSendMessage

File: app/src/main/res/layout/fragment_top.xml (hunk 6 above)
Comment (DavitTsiskarashvili) [reply app/src/main/res/layout/fragment_top.xml]: done

**Review by GiorgiMamagulashvil1 — APPROVED** [review]

**Review by AnaRusadze — APPROVED** [review]

**Review by Gio481 — APPROVED** [review]

**Pull request merged**